### ISSUE TITLE
Puppet-lint does not report hash rocket alignment inside a hash definition

### ISSUE THREAD

**Issue by asselvakumar** [rodjek/puppet-lint#951]
The `puppet-lint` does not report the hash rocket alignment warning inside the hash definition, for example.

```
$stools_ensure = $facts['fqdn'] ? {
    /^\w+alignment-us+/ => 'installed',
    default      => 'absent'
  }
```

If the same alignment is given inside the resource, `puppet-lint` is producing a warning, for example.

```
file { '/etc/passwd':
  ensure => file,
owner => 'root',
group => $rootgroup,
}
```

```
warning: puppet-lint: manifests/testing.pp:14:7: indentation of => is not properly aligned (expected in column 8, but found it in column 7)
```

**Comment by alexjfisher** [comment]
@asselvakumar Development has recently moved to a fork maintained by Puppet Inc.  You may wish to open this issue over at https://github.com/puppetlabs/puppet-lint/issues
